### ISSUE TITLE
[qgroundcontrol] Please update sha256sums

### ISSUE THREAD

**Issue by kikislater** [acxz/pkgbuilds#106]
hello sha256sum is wrong for qgroundcontrol-4.1.0.tar.gz : 

```
$ makepkg -fsi
==> Making package: qgroundcontrol 4.1.0-2 (mer. 27 janv. 2021 13:39:05)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found qgroundcontrol-4.1.0.tar.gz
  -> Found gps-drivers-qgc4.1.0.tar.gz
  -> Found mavlink-v2.0-qgc4.1.0.tar.gz
  -> Found aossl-qgc4.1.0.tar.gz
  -> Found gst-plugins-good-qgc4.1.0.tar.gz
  -> Found libicudata-qgc.patch
==> Validating source files with sha256sums...
    qgroundcontrol-4.1.0.tar.gz ... FAILED
    gps-drivers-qgc4.1.0.tar.gz ... Passed
    mavlink-v2.0-qgc4.1.0.tar.gz ... Passed
    aossl-qgc4.1.0.tar.gz ... Passed
    gst-plugins-good-qgc4.1.0.tar.gz ... Passed
    libicudata-qgc.patch ... Skipped
==> ERROR: One or more files did not pass the validity check!
```

**Comment by kikislater** [comment]
Correct one is : 
```
9a9c67e293290e296aa6716441f8ff5b7d9006286e2a6d8c145e283eab75e310
```

**Comment by acxz** [comment]
Whoa looks like they changed the release again ;(

**Comment by acxz** [comment]
Closed with #107
